code/lightna/magento-os-frontend/js/product/Options.js

```diff
@@ -1,13 +1,24 @@
-import { $, $$ } from '../lib/utils';
+import { $, $$, getBlockHtml } from '../lib/utils';
+import { UserInput } from '../lib/UserInput';
 
 export class ProductOptions {
+    cjs = '.cjs-product-options';
+    block = '.body.main.content.main.cta.add-to-cart.options';
```

Comment:
cons of solution is that once block moved in layout it won't work, right?

Reply:
yes, but we also have pros of solution, for example no JS  and no rendering same on SSR and Client :)
To make it working on client we will need to pass more json data, implement logic to determine avl of option and render by js same as it's rendered on the server

Reply:
btw, we can solve this cons by adding a possibility to bind id to the block in the layout. then render it by id instead of path.
it could also help to find whre it's used in js if you see block id

Reply:
will add blockid in another pull